samples/place-nearby-search/index.ts

```diff
@@ -5,72 +5,120 @@
  */
 
 // [START maps_place_nearby_search]
+const mapElement = document.querySelector('gmp-map') as google.maps.MapElement;
 let map;
+const advancedMarkerElement = document.querySelector('gmp-advanced-marker') as google.maps.marker.AdvancedMarkerElement;
+let center;
+let typeSelect;
+let infoWindow;
 
 async function initMap() {
     const { Map, InfoWindow } = await google.maps.importLibrary('maps') as google.maps.MapsLibrary;
+    const {LatLng} = await google.maps.importLibrary("core") as google.maps.CoreLibrary;
 
-    let center = new google.maps.LatLng(52.369358, 4.889258);
+    center = new LatLng(45.438646, 12.327573); // Venice, Italy
```

Comment:
is this line needed anymore? or can we pull this from the center property of the map, to dedupe from the html?